packages/core/ios/Sources/DdSdk.mm

```diff
@@ -79,7 +81,7 @@ @implementation DdSdk
 - (DdSdkImplementation*)ddSdkImplementation
 {
     if (_ddSdkImplementation == nil) {
-        _ddSdkImplementation = [[DdSdkImplementation alloc] init];
+        _ddSdkImplementation = [[DdSdkImplementation alloc] initWithBridge:_bridge];
```

Comment:
This is the actual fix, we're getting the bridge here (line 17) as we can do it  because this is the RN module.
Then we're passing the bridge to the DdSdkImplementation